index.html

```diff
@@ -662,62 +666,78 @@ <h3 class="bannerbox__title">Newsletter</h3>
-                        console.error(err);
-                    }
-                    availTimesObjectArray.push(outputDateTime);
+                    inputElem = $(element).find("input");
+
+                    startT = inputElem.attr('data-start');
+                    endT = inputElem.attr('data-end');
+
+                    availTimesObjectArray.push(startT + '/' + endT);
+
+                    // var outputDateTime = {};
+                    // try {
+                    //     var splitText = $(element).text().split(', ');
+                    //     //outputDateTime.month = splitText[1].split(' ')[0];
+                    //     //outputDateTime.day = splitText[1].split(' ')[1];
+                    //     var isoDate = new Date(splitText[1]);
+                    //     isoDate.setFullYear(2016); // HACK!!!!  Specific to 2016 Elections!
+
+                    //     var times = splitText[2].split(' ')[0];
+                    //     var startTime = times.split('-')[0];
+                    //     var endTime = times.split('-')[1];
+                    //     // Handle am/pm putting the time in the afternoon, if needed.
+                    //     var startHour = parseInt(startTime.split(':')[0]);
+                    //     if (startTime.indexOf('pm', 0) !== -1) {
+                    //         startHour = startHour + 12;
+                    //     }
+                    //     var startMinute = parseInt(startTime.split(':')[1].replace(/am/gi, '').replace(/pm/gi, ''));
+
+                    //     // Handle am/pm putting the time in the afternoon, if needed.
+                    //     var endHour = parseInt(endTime.split(':')[0]);
+                    //     if (endTime.indexOf('pm', 0) !== -1) {
+                    //         endHour = endHour + 12;
+                    //     }
+                    //     var endMinute = parseInt(endTime.split(':')[1].replace(/am/gi, '').replace(/pm/gi, ''));
+
+                    //     var startDateTime = new Date(isoDate); // Both Start and End are on same date.
+                    //     var endDateTime = new Date(isoDate); // Both Start and End are on same date.
+
+                    //     startDateTime.setHours(startHour);
+                    //     startDateTime.setMinutes(startMinute);
+                    //     endDateTime.setHours(endHour);
+                    //     endDateTime.setMinutes(endMinute);
+
+                    //     var startDateTimeISO = startDateTime.toISOString();
+                    //     var endDateTimeISO = endDateTime.toISOString();
+
+                    //     outputDateTime.startDttm = startDateTimeISO;
+                    //     outputDateTime.endDttm = endDateTimeISO;
+
+                    //     console.log("Date Time pair:" + JSON.stringify(outputDateTime));
+
+                    // } catch (err) {
+                    //     console.error(err);
+                    // }
+                    // availTimesObjectArray.push(outputDateTime);
```

Comment:
No need to keep this code commented if it's in the Git history, so we can just delete it, yeah?